### DISCUSSION TITLE
Cant Sync Past Block 259495 1.1.5

### DISCUSSION THREAD

**Discussion by TheWildJarvi** [Chia-Network/chia-blockchain#4487] [Support]
My chia gui app (1.1.5) wont sync past block 259495. Is this an error on my end or is something not working properly?
It was syncing up untill about 1pm EDT
![image](https://user-images.githubusercontent.com/44104303/117722495-24386180-b1af-11eb-9028-39a3fb32d5cb.png)

Thanks 
-
Jarvi

**Comment by lvcivs** [comment]
How many connected peers do you have? And what is their height? (it is shown in the GUI)

It's possible that you just need to give it more time.

**Reply by TheWildJarvi** [reply]
why would it randomly stop syncing tho? It was running all night. I restarted like 10 times today and changed upnp to being off.

**Comment by TheWildJarvi** [comment]
I restarted and opened logs and found this: 
![image](https://user-images.githubusercontent.com/44104303/117727601-408bcc80-b1b6-11eb-8cc9-6bbeef03cc8e.png)

**Comment by TheWildJarvi** [comment]
[debug.log.txt](https://github.com/Chia-Network/chia-blockchain/files/6455193/debug.log.txt)
Heres my latest log file after clearing it and running chia. Log is converted to .txt for upload

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
